### DISCUSSION TITLE
el-cascader开启filterable时多选时勾选有问题

### DISCUSSION THREAD

**Discussion by mtz0120** [element-plus/element-plus#17238] [Q&A]
<el-cascader
      v-model="test"
      placeholder="Try searchingL Guide"
      :options="options"
      :props="props"
      filterable
     @change="handleCascaderChange"
   />

当过滤搜索时，选中匹配项无法实现多选，且handleCascaderChange只执行一次，再次勾选不再执行，初次点击也无法勾选

**Comment by liuyao64** [comment]
遇到相同问题，请问你解决了吗
